docs/guide/mocking.md

```diff
@@ -673,14 +700,22 @@ test('can feed dogs', () => {
 ```
 :::
 
-Now, when we create a new instance of the `Dog` class its `speak` method (alongside `feed`) is already mocked:
+Now, when we create a new instance of the `Dog` class its `speak` method (alongside `feed` and `greet`) is already mocked:
 
 ```ts
 const dog = new Dog('Cooper')
 dog.speak() // loud bark!
+dog.greet() // Hi! My name is Cooper!
 
 // you can use built-in assertions to check the validity of the call
 expect(dog.speak).toHaveBeenCalled()
+expect(dog.greet).toHaveBeenCalled()
+
+const dog2 = new Dog('Max')
+
+// methods assigned to the prototype are shared between instances
+expect(dog2.speak).toHaveBeenCalled()
+expect(dog2.greet).not.toHaveBeenCalled()
```

Comment:
@sheremet-va 
I added this extra part to extend the example, going it's helpful